src/services/CalculatorProcess.js

```diff
@@ -1,51 +1,52 @@
-let memory = [];
-let result = "";
-function calculatorProcess(content,visor) {
-	
-	const numbersOperators = () => String(eval((result)));
-	
-	memory += content;
+let memory = "";
+let sLength;
 
-	if (memory.includes("=")) {
+function CalculatorProcess(content, visor) {
 
-		result = memory.slice(0,-1);
+	const firstInteraction = ["%", "/", "*", "+", "-", "S", "C", "="];
 
-		if (memory.includes("+")) {
-			memory = numbersOperators();
-		}
+	const conversion = (string) => String(eval(string));
 
-		if (memory.includes("-")) {
-			memory = numbersOperators();
-		}
+	memory += content;
 
-		if (memory.includes("*")) {
-			memory = numbersOperators();
-		}
+	if ((firstInteraction.includes(memory[0]) && memory.length < 2) || memory.includes("C")) {
+		memory = "";
 
-		if (memory.includes("/")) {
-			memory = numbersOperators();
-		}
+	}
+
+	if (memory[0] == ".") {
```

Comment:
what happen if I digit

```
1
.
5
```

Reply:
it'll show 1.5, this is for when the first digit it's a "." and it'll return by default "0." because don't exist,for example:  ".5" or ".5+.5=1.0".